Sources/Controllers/TargetMenu/POI/OAPOIViewController.mm

```diff
@@ -157,19 +158,42 @@ - (NSAttributedString *) getAdditionalInfoStr
         NSMutableAttributedString *str = [[NSMutableAttributedString alloc] init];
         UIColor *colorOpen = UIColorFromRGB(color_ctx_menu_amenity_opened_text);
         UIColor *colorClosed = UIColorFromRGB(color_ctx_menu_amenity_closed_text);
-        for (auto info : _openingHoursInfo)
+        for (int i = 0; i < _openingHoursInfo.size(); i ++)
         {
+            auto info = _openingHoursInfo[i];
+
             if (str.length > 0)
```

Comment:
This should be done using NSDateFormatter (see https://developer.apple.com/documentation/foundation/nsdateformatter/1409506-amsymbol)